habitat-lab/habitat/tasks/nav/nav.py

```diff
@@ -1631,6 +1659,14 @@ def _preprocess_action(self, xyt_waypoint, max_duration):
                     self._waypoint_ang_range[1],
                 ],
             )
+            delta_camera_pitch_angle = self._scale_inputs(
+                delta_camera_pitch_angle,
+                [0, 1],
```

Comment:
I think this should -1 to 1 since your action space is between that?
```suggestion
                [1, 1],
```